.env.development

```diff
@@ -0,0 +1,11 @@
+# Define ENV variables for development environment
+DATABASE_URL="postgresql://localhost/ruby_jobs_development"
```

Comment:
hey, could you remove this file from git?

Reply:
That's done